src/oscar/static_src/oscar/js/oscar/dashboard.js

```diff
@@ -264,11 +264,16 @@ var oscar = (function(o, $) {
                 }
             });
 
+            // Add href to url, so when the page is reloaded this tab will be displayed.
```

Comment:
Just a quick comment here; In the past jquery would do this, but jquery deprecated this.